common/crypto_utils/setup.py

```diff
@@ -32,85 +32,13 @@
 version = subprocess.check_output(
     os.path.join(tcf_root_dir, 'bin/get_version')).decode('ascii').strip()
 
-openssl_cflags = subprocess.check_output(
-    ['pkg-config', 'openssl', '--cflags']).decode('ascii').strip().split()
-openssl_include_dirs = list(
-    filter(None, re.split('\s*-I',
-           subprocess.check_output(
-               ['pkg-config', 'openssl', '--cflags-only-I'])
-               .decode('ascii').strip())))
-openssl_libs = list(
-    filter(None, re.split('\s*-l',
-           subprocess.check_output(['pkg-config', 'openssl', '--libs-only-l'])
-           .decode('ascii').strip())))
-openssl_lib_dirs = list(
-    filter(None, re.split('\s*-L',
-           subprocess.check_output(['pkg-config', 'openssl', '--libs-only-L'])
-           .decode('ascii').strip())))
-
 compile_args = [
     '-std=c++11',
     '-Wno-switch',
     '-Wno-unused-function',
     '-Wno-unused-variable',
 ]
 
-
-crypto_include_dirs = [
-    os.path.join(tcf_root_dir, 'common/cpp'),
-    os.path.join(tcf_root_dir, 'common/cpp/crypto'),
-] + openssl_include_dirs
-
-verify_report_include_dirs = [
-    os.path.join(tcf_root_dir, 'common/cpp'),
-    os.path.join(tcf_root_dir, 'common/cpp/crypto'),
-    os.path.join(tcf_root_dir, 'common/cpp/packages/parson'),
-    os.path.join(tcf_root_dir, 'common/cpp/packages/base64'),
-    os.path.join(tcf_root_dir, 'common/cpp/verify_ias_report'),
-] + openssl_include_dirs
-
-library_dirs = [
-    os.path.join(tcf_root_dir, "common/cpp/build"),
-] + openssl_lib_dirs
-
-# Do not change the order of these
-# libraries otherwise we will get the undefined
-# symbols in verify_report shared library
-libraries = [
-    'uavalon-verify-ias-report',
-    'uavalon-crypto',
-    'uavalon-common',
-    'uavalon-base64',
-    'uavalon-parson',
-] + openssl_libs
-
-crypto_modulefiles = [
-    "avalon_crypto_utils/crypto/crypto.i",
-]
-
-crypto_module = Extension(
```

Comment:
import of `Extension` at top can also be removed

Reply:
Removed